Description: Set the locale when generating the Javadoc to make the API documentation reproducible
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/src/tools/org/h2/build/BuildBase.java
+++ b/src/tools/org/h2/build/BuildBase.java
@@ -524,6 +524,10 @@
      * @param args the command line arguments to pass
      */
     protected void javadoc(String...args) {
+        StringList argumentList = args(args);
+        argumentList.addAll(0, Arrays.asList("-locale", "en", "-encoding", "UTF-8"));
+        args = argumentList.array();
+
         int result;
         PrintStream old = System.out;
         try {
